src/root.zig

```diff
@@ -463,6 +487,13 @@ pub const AsciiOptions = struct {
 
     penalty_case_mistmatch: i32 = -2,
 
+    /// Don't forget the allocator !!!
+    allocator: Allocator = undefined,
```

Comment:
Let's pass allocators in from the algorithm struct instead of having the options hold on to them.

Reply:
Removed in commit https://github.com/alberic89/fuzzig/commit/3bab90bbb045f787695c80699736ad83483b6235
But many side effects. Maybe they will be attenuated with the futures changes (less need of an allocator).